source/adminguide/templates/_cloud_init.rst

```diff
@@ -62,6 +62,11 @@ These features can be implemented in `“Linux Template creation process” <_cr
      datasource:
        CloudStack: {}
        None: {}
+
+   .. note::
+      For the vm instances running on VMware or XenServer/XCP-ng hypervisors, if there are multiple cloud-init data sources, it is a known issue that ds-identify is not able to detect if "CloudStack" DataSource is enabled. To fix the problem, please run the following command to enable cloud-init without any aid from ds-identify.
+
+        echo "policy: enabled" >  /etc/cloud/ds-identify.cfg
```

Comment:
```suggestion
        .. code:: bash

          echo "policy: enabled" >  /etc/cloud/ds-identify.cfg
```

Reply:
thanks @vishesh92 
let's see if it works

Reply:
looks good

![image](https://github.com/apache/cloudstack-documentation/assets/57355700/f85e8e6b-d91d-4b89-8d87-711dedd24b9f)

https://cloudstack-documentation--383.org.readthedocs.build/en/383/adminguide/templates/_cloud_init.html